knack/iframeMapMessenger.js

```diff
@@ -76,12 +77,23 @@
       };
 
       sendMessageToApp(markerMessage, locationViewIFrame);
+    }
+    // Location Details Page - Editable
+    $("#view_2609 #mapIFrame").on("load", function() {
```

Comment:
Does this need to happen after the change to rendering the map in view_2733 in both Editable and Viewable Location Details views? I'm glad you made that switch btw since it makes the view consistent across roles. 👍